packages/use-dataloader/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@scaleway/use-dataloader",
+  "version": "1.0.0",
+  "description": "A small hook to handle api requests",
+  "keywords": [
+    "react",
+    "react-dom",
+    "reactjs",
+    "hooks",
+    "dataloader",
+    "axios",
+    "xhr"
+  ],
+  "main": "dist/index.js",
+  "module": "dist/module.js",
+  "browser": {
+    "dist/index.js": "./dist/index.browser.js",
+    "dist/module.js": "./dist/module.browser.js"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/scaleway/scaleway-lib",
+    "directory": "packages/use-dataloader"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "prop-types": "^15.7.2"
+  },
+  "peerDependencies": {
+    "react": "17.x",
+    "react-dom": "17.x"
```

Comment:
We have no implicit react-dom dependency right (i.e. we could use this package with react-native) ?

Reply:
I think yes I dropped it